### PR TITLE
fixing vdw_radii assignments for unexpected atom/particle names

### PR DESCRIPTION
**Pull request by rbdavid**
Directly associated with #484 and tangentially #452 and #468.

### CODE REVIEW

**Comment by rbdavid** [comment]
I'm gonna remove the UML diagrams that I had created for this branch of my fork. I was using it to visually guide my understanding of MN's organization. Not related to this PR.

**Comment by BradyAJohnston** [comment]
Apologies for delay in getting back to you. As mentioned elsewhere I'm away hiking at the moment so I can't review until
I'm back in 1-2 weeks

File: molecularnodes/io/parse/mda.py [hunk 1]
```diff
@@ -176,9 +176,8 @@ def atomic_number(self) -> np.ndarray:
     def vdw_radii(self) -> np.ndarray:
         # pm to Angstrom
         return np.array(
-            [data.elements.get(element,
-                               {'vdw_radii': 100})
-             .get('vdw_radii') for element in self.elements]) * 0.01 * self.world_scale
+            [data.elements.get(element,{}).get(
+                'vdw_radii',100) for element in self.elements]) * 0.01 * self.world_scale
```
Comment (BradyAJohnston) [inline molecularnodes/io/parse/mda.py:180]: This is maybe a matter for a different PR, but I was mildly confused why we had the ` * 0.01 * world_scale` before I remembered that we have the vdw_radii in picometres. Something that could probably be handled better in the future.

**Review by BradyAJohnston — APPROVED** [review]

**Comment by BradyAJohnston** [comment]
Thanks for fixing this!

**Pull request merged**